kde-plasma/plasma-workspace/metadata.xml

```diff
@@ -8,7 +8,7 @@
 	<use>
 		<flag name="appstream">Enable AppStream software metadata support</flag>
 		<flag name="geolocation">Enables dataengine providing location information</flag>
-		<flag name="qalculate">Enable Qalculate runner using <pkg>sci-libs/libqalculate</pkg></flag>
+		<flag name="qalculate">Replace krunner calculator QJSEngine engine with Qalculate using <pkg>sci-libs/libqalculate</pkg></flag>
```

Comment:
More standard phrasings would be `Use ... instead of ...`.
(See e.g. `gles2-only`, `graphicsmagick`, `libressl`, `llvm-libunwind` global USE flags).

So possible suggestions for this case:
```
<flag name="qalculate">Use <pkg>sci-libs/libqalculate</pkg> instead of QJSEngine (<pkg>dev-qt/qtdeclarative</pkg>) for krunner calculator</flag>
<flag name="qalculate">Use <pkg>sci-libs/libqalculate</pkg>-based engine instead of QJSEngine (<pkg>dev-qt/qtdeclarative</pkg>)-based engine for krunner calculator</flag>

Reply:
Thank you for the suggestions, I've went with the more concise version.